src/basic/missing_syscall.h

```diff
@@ -637,13 +637,63 @@ int __clone2(int (*fn)(void *), void *stack_base, size_t stack_size, int flags,
 #if !HAVE_QUOTACTL_FD
 
 static inline int missing_quotactl_fd(int fd, int cmd, int id, void *addr) {
-#if defined __NR_quotactl_fd
+#  ifdef __NR_quotactl_fd
         return syscall(__NR_quotactl_fd, fd, cmd, id, addr);
-#else
+#  else
         errno = ENOSYS;
         return -1;
-#endif
+#  endif
 }
 
 #  define quotactl_fd missing_quotactl_fd
 #endif
+
+/* ======================================================================= */
+
+#if !HAVE_SETXATTRAT
+struct xattr_args {
+        _align_(8) uint64_t value;
```

Comment:
uint64_t implies alignment to 8 byte anyway, so why the explicit align expression?

Reply:
I just copied the kernel definition verbatim here